app/gimletlet/app.toml

```diff
@@ -157,6 +157,20 @@ start = true
 task-slots = ["net"]
 features = ["vlan"]
 
+[tasks.mgmt_gateway]
+name = "task-mgmt-gateway"
+priority = 4
+max-sizes = {flash = 32768, ram = 8192}
+stacksize = 4096
+start = true
+uses = [
+    "usart1",
+    "system_flash", # TODO also used by `net`, both to read the stm32 uid
+]
+task-slots = ["net", "sys"]
+features = ["usart1", "vlan"]
+interrupts = {"usart1.irq" = 2}
```

Comment:
Since the interrupt notification masks are, well, masks, please write this in binary to make it slightly less likely to be confused with a bit _number_: `0b10`

Reply:
Fixed in 0c4ee9cf